.github/workflows/docsbuild-release.yaml

```
@@ -0,0 +1,73 @@
+name: ansible-release-porting-guide
+on:
+  workflow_dispatch:
+    inputs:
+      ansible_version:
+        description: 'Release Version. Example : 11.1.0'
+        required: true
+      ansible_major_version:
+        description: 'Exmaple 11'
+        required: true
+
+env:
```

Use `vars` context, instead of `env`. No need to expose these to every process.
```suggestion
vars:
```

---

Disregard this comment. Turns out, you can only define `vars` through the repo settings and not in the workflow definition.